### PR TITLE
Add StringTie 1.3.3

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,7 +14,8 @@ RUN apt-get update -y && apt-get install -y \
     python-dev \
     python-pip \
     unzip \
-    wget
+    wget \
+    tzdata
```
Comment (tmooney) [inline Dockerfile]: Might be nice to keep these alphabetized!

**Review by tmooney — APPROVED** [review]
+1 modulo my nitpicking!

File: Dockerfile (hunk 1 above)
Comment (susannasiebert) [reply Dockerfile]: I didn't realize they were. Will fix.

**Review by jasonwalker80 — APPROVED** [review]
+1

**Pull request merged**